meta-lxatac-bsp: linux-lxatac: upgrade to 6.19

hnez opened the pull request:
umpf tag is now: 6.19/customers/lxa/lxatac/20260210-1
The tag includes the following topic branches:

  - v6.11/topic/reproducible-build
  - v6.19/topic/m_can-reset

TODO before merging:

- [x] Test on hardware
- [x] Wait for 6.19 to be released

hnez committed:
meta-lxatac-bsp: linux-lxatac: upgrade to 6.19

umpf tag is now: 6.19/customers/lxa/lxatac/20260218-1
The tag includes the following topic branch:

  - v6.11/topic/reproducible-build

The following topic branch was dropped, because the patch it contains
was dropped upstream:

  - v6.19/topic/m_can-reset

Signed-off-by: Leonard Göhrs <l.goehrs@pengutronix.de>

hnez committed:
meta-lxatac-bsp: linux-lxatac: enable RTW89 based USB WiFi dongles

Signed-off-by: Leonard Göhrs <l.goehrs@pengutronix.de>

hnez commented:
It is likely that we will need this RAUC PR applied as patch in order to perform RAUC updates on a running 6.19 system: https://github.com/rauc/rauc/pull/1846

hnez commented:
Who would be willing to review this change?

hnez commented:
The RAUC patch has landed in RAUC 1.15.1 which is available in our current version of meta-rauc, so I have removed it here.

jluebbe commented:
@marckleinebudde mentions that he [dropped the remaining m_can patch](https://lore.kernel.org/all/20250910-strange-hopeful-chamois-6c4b6f-mkl@pengutronix.de/) and recommends dropping it here as well.

hnez commented:
> @marckleinebudde mentions that he [dropped the remaining m_can patch](https://lore.kernel.org/all/20250910-strange-hopeful-chamois-6c4b6f-mkl@pengutronix.de/) and recommends dropping it here as well.

Thanks for noting this. I have dropped the patch and rebased this PR while at it.

jluebbe reviewed (approved):
Looks good to me.

marckleinebudde reviewed (approved):
LGTM
